### ISSUE TITLE
Hrana disconnects with long lived streams

### ISSUE THREAD

**Issue by exaby73** [simolus3/hrana.dart#5]
I'm facing the following errors:

- https://github.com/tursodatabase/libsql/issues/985
- https://github.com/tursodatabase/libsql-client-ts/issues/119

A sample log from my Dart application:

```console
HranaException(400): Failed to run pipeline: "Received a baton that has already been used"
#0      HranaHttpClient._runPipeline (package:hrana/src/rpc_http_client.dart:84:7)
<asynchronous suspension>
simolus3/drift#1      HranaHttpClient.executeStatement (package:hrana/src/rpc_http_client.dart:161:22)
<asynchronous suspension>
simolus3/drift#2      Database._runSql (package:hrana/src/database.dart:78:12)
<asynchronous suspension>
simolus3/drift#3      Database.select (package:hrana/src/database.dart:92:22)
<asynchronous suspension>
simolus3/drift#4      _HranaDelegate.runSelect (package:drift_hrana/drift_hrana.dart:40:17)
<asynchronous suspension>
simolus3/drift#5      _BaseExecutor.runSelect (package:drift/src/runtime/executor/helpers/engines.dart:74:20)
<asynchronous suspension>
simolus3/drift#6      InsertStatement.insertReturningOrNull.<anonymous closure> (package:drift/src/runtime/query_builder/statements/insert.dart:174:22)
<asynchronous suspension>
simolus3/drift#7      InsertStatement.insertReturning (package:drift/src/runtime/query_builder/statements/insert.dart:152:9)
```

**Comment by simolus3** [comment]
I see that there's a fairly short timeout of 10 seconds for stream ids on the server... So we should probably not be using that for the entire drift database haha.

**Comment by simolus3** [comment]
This should be fixed in version `1.0.3` of `drift_hrana`. Please let me know if you still run into any issues with that version.

**Comment by exaby73** [comment]
Thank you so much!
